src/name_bazaar/ui/events/public_resolver_events.cljs

```diff
@@ -69,3 +67,52 @@
      {:db (assoc-in db [:public-resolver/reverse-records
                         addr
                         :public-resolver.record/name] name)})))
+
+(reg-event-fx
+ :public-resolver.name/point
+ [interceptors (validate-first-arg (s/keys :req [:ens.record/name
+                                                 :ens.record/address]))]
```

Comment:
`:ens.record/address` should be called `:public-resolver.record/addr`, to reflect smart contracts. ENS record has no property `address`

Reply:
Maybe we should stop using namespaces for passed params to events, as it's not always clear what is what.